### DISCUSSION TITLE
Make addresses multiline text fields

### DISCUSSION THREAD

**Discussion by Moini** [baimard/gestion#187] [Q&A]
Addresses can currently only be on a single line. This is pretty uncommon. 

Please allow entering address information with multiple lines, by providing a text area instead of a text field.

**Comment by baimard** [comment]
This is not a bug, this need to be open in a new discussion : https://github.com/baimard/gestion/discussions

Also, to prevent a duplicate discussion, please take the time to read all the opens discussions.

**Reply by jblb** [reply]
same request here, most adresses are multilines

**Comment by Moini** [comment]
See https://github.com/github/feedback/discussions/2952 for how to convert an issue into a discussion. Really, you can also use a tag 'feature request'. This is what people commonly do.

It is a bug for me, though, as in Germany, we write addresses into multiple lines. It looks very odd to have to put them on a single line, especially for long addresses, like institutions, universities etc.

**Comment by baimard** [comment]
I understand, I need to take some modification to allow the possibility to insert html. But actually it's not a priority.

**Reply by Moini** [reply]
Neither it is for me :) I've only tried it out and found it not to be useful to me in its current state. 

The comma issues especially (no comma allowed in unit price - gives error, but not fatal, just sets the price to 0 on reload; no comma for number of products allowed - just rounds 1.5 working hours to 2...) are blockers for using it at all. 

So I went here to report my findings, and might try it out again in the future, as well as I'm going to contribute to kraft. 

Whichever one works well enough first is going to be the thing that I'll use for production. Until then, everything is going to stay manual work with office templates.

**Reply by baimard** [reply]
> The comma issues especially (no comma allowed in unit price - gives error, but not fatal, just sets the price to 0 on reload; no comma for number of products allowed - just rounds 1.5 working hours to 2...) are blockers for using it at all.

it's because you need in version 2.0.4 a dot in place of commat, I have solved that in 2.0.5 (I just push the new version)

It's because float in database work with dot (english format).

Please consider that a very young application.

**Reply by Moini** [reply]
Yes, I know!

Would using a number input field help? `<input type="number" step="0.01"> ` - that should display correctly for the user, in the user's locale, and give you back a value that already is a float.

**Reply by baimard** [reply]
That's a good idea that I hadn't thought of. The importance of talking with other developers...

I solved the problem differently. Because there was also the monetary sign to integrate.

Thank you !

**Reply by baimard** [reply]
it's done !

**Reply by LiventNick** [reply]
How do you make a multi-line? I am currently trying to figure it out because its not working and in Canada our addresses are multilines
123 Example Street
City, Province
Postal Code
(if international) CANADA
